python/cuml/test/test_fil.py

```diff
@@ -179,59 +239,69 @@ def test_fil_skl_classification(n_rows, n_columns, n_estimators, max_depth,
     fil_preds = np.asarray(fm.predict(X_validation))
     fil_acc = accuracy_score(y_validation, fil_preds)
 
-    print("SKL accuracy = ", skl_acc, " ForestInference accuracy: ", fil_acc)
+    print("SKL accuracy = ", skl_acc, " FIL accuracy: ", fil_acc)
     assert fil_acc == skl_acc
```

Comment:
I think the second check is probably sufficient, or maybe the goal here is to see whether we are close or very far off when we fail? I guess I'm fine with that too

Reply:
Actually, `array_equal` is not an exact comparison (see its definition for details). I've kept the accuracy comparison, but made it approximate.